catalog/etcd/catalog.bom

```diff
@@ -0,0 +1,37 @@
+brooklyn.catalog:
+  items:
+    - classpath://io.brooklyn.etcd.brooklyn-etcd:etcd/etcd.bom
+
+    - id: etcd-cluster-template
+      name: "etcd-cluster"
+      iconUrl: https://raw.githubusercontent.com/coreos/etcd/master/logos/etcd-stacked-color.png
```

Comment:
Can this file be copied to `resources/icons/etcd.png` in the repo, and referenced as a `classpath://` resource instead, to allow use when github access is not possible.